### PR TITLE
Fix translator comments [Fixes #402]

### LINKED ISSUE #402: Add missing translator comments to strings with variables
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/class-google-analytics.php#L323
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/class-google-analytics.php#L336
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/class-google-analytics.php#L349
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/dashboard.php#L43
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/dashboard.php#L52
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/dashboard.php#L65
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/dashboard.php#L82
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/dashboard.php#L91
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/dashboard.php#L111
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/class-admin.php#L322
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L101
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L110
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L118 (translator comment is not on the line before the gettext function)
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L124
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L138
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L140
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L144
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L145
* [x] https://github.com/Yoast/google-analytics-for-wordpress/blob/trunk/admin/pages/settings.php#L146

### PR DESCRIPTION
**Pull request by Petervw**
Fixes #402

### CODE REVIEW

File: admin/class-admin.php [hunk 1]
```diff
@@ -329,6 +329,7 @@ public function add_action_links( $links ) {
 	public function premium_promo() {
 		echo '<div class="ga-promote">';
 		echo '<p>';
+		/* translators: %s is a link to the Google Analytics product page on Yoast.com */
```
Comment (andizer) [inline admin/class-admin.php]: `%s` have to be `%1$s` and the `%2$s` is missing.

File: admin/class-google-analytics.php [hunk 2]
```diff
@@ -319,6 +319,7 @@ class Yoast_Google_Analytics_Notice {
 	 * Throw a warning if no UA code is set.
 	 */
 	public static function config_warning() {
+		/* translators: %s is a link to the Google Analytics admin settings page */
 		self::show_error(
 			sprintf( __( 'Please configure your %sGoogle Analytics settings%s!', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/class-google-analytics.php]: Using `%1$s` and `%2$s` will be more readable. When replacing `%s` into those two the translator comment have to be changed as well.

File: admin/class-google-analytics.php [hunk 3]
```diff
@@ -331,6 +332,7 @@ public static function config_warning() {
 	 * Throw a warning when the fetching failed
 	 */
 	public static function warning_fetching_data_authenticate() {
+		/* translators: %s is a link to the Google Analytics admin settings page */
 		self::show_error(
 			sprintf(
 				__( 'It seems the authentication for the plugin has expired, please %sre-authenticate%s with Google Analytics to allow the plugin to fetch data.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/class-google-analytics.php]: Please use `%1$s` and `%2$s`

File: admin/class-google-analytics.php [hunk 4]
```diff
@@ -344,6 +346,7 @@ public static function warning_fetching_data_authenticate() {
 	 * Throw a warning when the fetching failed
 	 */
 	public static function warning_fetching_data() {
+		/* translators: %s is a link to an article on our KB */
 		self::show_error(
 			sprintf(
 				__( 'Data is not up-to-date, there was an error in retrieving the data from Google Analytics. This error could be caused by several issues. If the error persists, please see %sthis page%s.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/class-google-analytics.php]: Please use `%1$s` and `%2$s`

File: admin/pages/dashboard.php [hunk 5]
```diff
@@ -39,6 +39,7 @@ function (num, element) {
 				if ( $tracking_code !== '' ) {
 					if ( empty( $options['analytics_profile'] ) ) {
 						echo '<div class="ga-promote"><p>';
+						/* translators: %s is a link to the Google Analytics admin settings page */
 						echo sprintf(
 							__( 'We need you to authenticate with Google Analytics to use this functionality. If you set your UA-code manually, this won\'t work. You can %sauthenticate your Google Analytics profile here%s to enable dashboards.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/pages/dashboard.php]: Please use `%1$s` and `%2$s`

File: admin/pages/dashboard.php [hunk 6]
```diff
@@ -48,6 +49,7 @@ function (num, element) {
 					}
 					else if ( ! Yoast_Google_Analytics::get_instance()->has_refresh_token() ) {
 						echo '<div class="ga-promote"><p>';
+						/* translators: %s is a link to the Google Analytics admin settings page */
 						echo sprintf(
 							__( 'Because we\'ve switched to a newer version of the Google Analytics API, you\'ll need to re-authenticate with Google Analytics. We\'re sorry for the inconvenience. You can %sre-authenticate your Google Analytics profile here%s.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/pages/dashboard.php]: Please use `%1$s` and `%2$s`

File: admin/pages/dashboard.php [hunk 7]
```diff
@@ -61,6 +63,7 @@ function (num, element) {
 				}
 				else {
 					echo '<div class="ga-promote"><p>';
+					/* translators: %s is a link to the Google Analytics admin settings page */
 					echo sprintf(
 						__( 'You have not yet finished setting up Google Analytics for Wordpress by Yoast. Please %sadd your Analytics profile here%s to enable tracking.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/pages/dashboard.php]: Please use `%1$s` and `%2$s`

File: admin/pages/dashboard.php [hunk 8]
```diff
@@ -78,6 +81,7 @@ function (num, element) {
 			if ( $tracking_code !== '' ) {
 				if ( empty( $options['analytics_profile'] ) ) {
 					echo '<div class="ga-promote"><p>';
+					/* translators: %s is a link to the Google Analytics admin settings page */
 					echo sprintf(
 						__( 'We need you to authenticate with Google Analytics to use this functionality. If you set your UA-code manually, this won\'t work. You can %sauthenticate your Google Analytics profile here%s to enable dashboards.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/pages/dashboard.php]: Please use `%1$s` and `%2$s`

File: admin/pages/dashboard.php [hunk 9]
```diff
@@ -87,6 +91,7 @@ function (num, element) {
 				}
 				else if ( ! Yoast_Google_Analytics::get_instance()->has_refresh_token() ) {
 					echo '<div class="ga-promote"><p>';
+					/* translators: %s is a link to the Google Analytics admin settings page */
 					echo sprintf(
 						__( 'Because we\'ve switched to a newer version of the Google Analytics API, you\'ll need to re-authenticate with Google Analytics. We\'re sorry for the inconvenience. You can %sre-authenticate your Google Analytics profile here%s.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/pages/dashboard.php]: Please use `%1$s` and `%2$s`

File: admin/pages/dashboard.php [hunk 10]
```diff
@@ -107,6 +112,7 @@ function (num, element) {
 			}
 			else {
 				echo '<div class="ga-promote"><p>';
+				/* translators: %s is a link to the Google Analytics admin settings page */
 				echo sprintf(
 					__( 'You have not yet finished setting up Google Analytics for Wordpress by Yoast. Please %sadd your Analytics profile here%s to enable tracking.', 'google-analytics-for-wordpress' ),
```
Comment (andizer) [inline admin/pages/dashboard.php]: Please use `%1$s` and `%2$s`

**Comment by CarolineGeven** [comment]
Processed @andizer's feedback. Also removed 'Google Analytics by Yoast' and 'Google Analytics by Yoast Premium' from translatable strings.

**Comment by boblinthorst** [comment]
Why is there a minified .js file without the corresponding js file?

File: admin/pages/settings.php [hunk 11]
```diff
@@ -135,15 +138,20 @@
...
+		/* translators: 1: link to the Google Analytics help page on Google.com to enable link attribution. 2: link closing tag */
 		echo Yoast_GA_Admin_Form::input( 'checkbox', __( 'Allow anchor', 'google-analytics-for-wordpress' ), 'allow_anchor', null, sprintf( __( 'This adds a %1$s call to your tracking code, and makes RSS link tagging use a %2$s as well.', 'google-analytics-for-wordpress' ), '<a href="https://developers.google.com/analytics/devguides/collection/gajs/methods/gaJSApiCampaignTracking?csw=1#_gat.GA_Tracker_._setAllowAnchor" target="_blank"><code>_setAllowAnchor</code></a>', '<code>#</code>' ) );
+		/* translators: 1: link to the Google Analytics help page on Google.com add the allow linker tag. 2: link closing tag */
```
Comment (boblinthorst) [inline admin/pages/settings.php]: Second part of the translation comment is not a closing tag, but a code tag.

File: admin/pages/settings.php [hunk 12]
```diff
@@ -135,15 +138,20 @@
 		echo Yoast_GA_Admin_Form::select( __( 'Track downloads as', 'google-analytics-for-wordpress' ), 'track_download_as', $yoast_ga_admin->track_download_types(), __( 'Not recommended, as this would skew your statistics, but it does make it possible to track downloads as goals.', 'google-analytics-for-wordpress' ) );
 		echo Yoast_GA_Admin_Form::input( 'text', __( 'Extensions of files to track as downloads', 'google-analytics-for-wordpress' ), 'extensions_of_files', null, 'Please separate extensions using commas' );
 		echo Yoast_GA_Admin_Form::select( __( 'Track full URL of outbound clicks or just the domain', 'google-analytics-for-wordpress' ), 'track_full_url', $yoast_ga_admin->get_track_full_url() );
+		/* translators: 1: link to the Google Analytics help page on Google.com to set subdomain tracking. 2: link closing tag */
```
Comment (boblinthorst) [inline admin/pages/settings.php]: Claims there are two translator comment parts, Only one in the implementation. second one can be removed.

File: admin/pages/settings.php [hunk 13]
```diff
@@ -135,15 +138,20 @@
...
 		echo Yoast_GA_Admin_Form::input( 'text', __( 'Label for those links', 'google-analytics-for-wordpress' ), 'track_internal_as_label', null, 'The label to use for these links, this will be added to where the click came from, so if the label is "aff", the label for a click from the content of an article becomes "outbound-article-aff".' );
 
 		echo Yoast_GA_Admin_Form::input( 'checkbox', __( 'Tag links in RSS feed with campaign variables', 'google-analytics-for-wordpress' ), 'tag_links_in_rss', null, __( 'Do not use this feature if you use FeedBurner, as FeedBurner can do this automatically and better than this plugin can. Check <a href="https://support.google.com/feedburner/answer/165769?hl=en&amp;ref_topic=13075" target="_blank">this help page</a> for info on how to enable this feature in FeedBurner.', 'google-analytics-for-wordpress' ) );
+		/* translators: 1: link to the Google Analytics help page on Google.com to enable link attribution. 2: link closing tag */
```
Comment (boblinthorst) [inline admin/pages/settings.php]: Second part of the translation comment is not a closing tag, but a code tag.

File: admin/pages/settings.php [hunk 14]
```diff
@@ -135,15 +138,20 @@
...
+		/* translators: 1: link to the Google Analytics help page on Google.com add the allow linker tag. 2: link closing tag */
 		echo Yoast_GA_Admin_Form::input( 'checkbox', __( 'Add <code>_setAllowLinker</code>', 'google-analytics-for-wordpress' ), 'add_allow_linker', null, sprintf( __( 'This adds a %1$s call to your tracking code, allowing you to use %2$s and related functions.', 'google-analytics-for-wordpress' ), '<a href="https://developers.google.com/analytics/devguides/collection/gajs/methods/gaJSApiDomainDirectory?csw=1#_gat.GA_Tracker_._setAllowLinker" target="_blank"><code>_setAllowLinker</code></a>', ' <code>_link</code>' ) );
+
 		if ( current_user_can( 'unfiltered_html' ) ) {
+			/* translators: 1: is a link to the Google Analytics help page on Google.com to add custom code for the tracking, 2: link closing tag */
```
Comment (boblinthorst) [inline admin/pages/settings.php]: Claims there are two translator comment parts, Only one in the implementation. second one can be removed.

**Comment by boblinthorst** [comment]
Code Review done.

**Pull request merged**